packages/text-input/TextInput.js

```diff
@@ -30,20 +30,17 @@ export default function TextInput({ html, state }) {
   return html`
 <style>
   :host label input {
-    background-color: var(--light);
-    border-color: var(--grey-300);
+    background-color: var(--back);
+    border-color: var(--fore);
   }
   :host label input:focus {
-    outline: none;
-    border-color: var(--grey-700);
-    transition: border-color 0.15s ease-in;
+    transition: outline 0.15s ease-in;
   }
   :host .errors {
-    color: var(--error-500);
-    border-color: var(--error-500);
+    color: var(--error);
```

Comment:
aXe and Lighthouse report contrast issues with this color in light and dark mode.

Reply:
These values are directly from the Apple human interface guides. I will pick some higher contrast colors.